pkg/apis/agones/v1/gameserver_test.go

```diff
@@ -1253,7 +1253,7 @@ func TestGameServerPod_ContainerNotFound_ErrReturned(t *testing.T) {
 
 }
 
-func TestGameServerPod_WithSidecar_NoErrros(t *testing.T) {
+func TestGameServerPod_WithSidecar_NoErrors(t *testing.T) {
```

Comment:
Somehow this slipped in, and I missed it - can we get rid of the `_`, while we are in here please :D

It's not to Go naming conventions.

Reply:
By the way underscore is fine for Example functions in tests, probably that's why `golint` did not show an error:
https://golang.org/pkg/testing/#hdr-Examples